doc/page-annotations.h

```diff
@@ -85,4 +85,11 @@ struct MyStruct
 }
 \endcode
 
+\section Annotations_interactions Interactions
+All annotations currently should have no interactions between each other.
```

Comment:
the use of "should" suggests that you are not 100% sure. This is confirmed by the statement below.
The documentation should be clear.